src/Microsoft.ML.Tokenizers/Normalizer/SentencePieceNormalizer.cs

```diff
@@ -386,7 +386,7 @@ internal int Normalize(ReadOnlySpan<byte> input, ref Span<byte> normalized, ref
             while (!input.IsEmpty)
             {
                 int p = NormalizePrefix(input, out Memory<byte> normalizedPrefix);
-                ReadOnlySpan<byte> sp = normalizedPrefix.Equals(default(Memory<byte>)) ? input.Slice(0, p) : normalizedPrefix.Span;
+                ReadOnlySpan<byte> sp = normalizedPrefix.Length == 0 ? input.Slice(0, p) : normalizedPrefix.Span;
```

Comment:
Makes sense - I looked for other instances https://github.com/search?q=repo%3Adotnet%2Fmachinelearning+%2FEquals%5C%28default%5C%28Memory%2F&type=code this is it.